tests/integration/integration_python_acceptance_test.go

```diff
@@ -565,4 +565,18 @@ in-project = true`
 
 	require.True(t, e.PathExists("pyproject.toml"), "pyproject.toml was created")
 	require.False(t, e.PathExists("requirements.txt"), "requirements.txt was removed")
+
+	b, err := os.ReadFile(filepath.Join(e.RootPath, "pyproject.toml"))
+	require.NoError(t, err)
+	require.Equal(t, `[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool]
```

Comment:
Bit odd it writes out an empty section?

Reply:
I think it's because it's the parent of "tool.poetry". Both github.com/pelletier/go-toml/v2 and github.com/BurntSushi/toml generate it.